### PR TITLE
Add supported architectures to all apps.

### PR DESCRIPTION
**Pull request by steveiliop56**

### CODE REVIEW

**Comment by steveiliop56** [comment]
@meienberger everything ready...

File: apps/__tests__/apps.test.ts [hunk 1]
```diff
@@ -137,6 +138,17 @@ describe("App configs", () => {
     });
   });
 
+  describe("Each app should have a supported architecture", () => {
+    const apps = getAppConfigs();
+
+    apps.forEach((app) => {
+      test(app.id, () => {
+        expect(app.supported_architectures).toBeDefined();
+        expect(app.categories).toBeInstanceOf(Array);
```
Comment (meienberger) [inline apps/__tests__/apps.test.ts:147]: Change categories to supported_architectures

File: apps/mongo-express/config.json [hunk 2]
```diff
@@ -13,6 +13,14 @@
   "source": "https://github.com/mongo-express/mongo-express",
   "website": "https://www.mongodb.com/",
   "form_fields": [
+    {
+      "type": "text",
```
Comment (meienberger) [inline apps/mongo-express/config.json:17]: Use type `ip` instead of text

File: apps/peppermint/docker-compose.arm64.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+version: "3.7"
+
+services:
+  peppermint:
+    image: pepperlabs/peppermint:dev
```
Comment (meienberger) [inline apps/peppermint/docker-compose.arm64.yml:5]: In config.json it states latest so we should use this one. If arm is only available in `dev` we should rather drop arm support

**Review by meienberger — CHANGES_REQUESTED** [review]

File: apps/peppermint/docker-compose.arm64.yml (hunk 3 above)
Comment (steveiliop56) [reply apps/peppermint/docker-compose.arm64.yml:5]: Its actually old so the dev would be preferred check the tags here https://hub.docker.com/r/pepperlabs/peppermint/tags.

Comment (meienberger) [reply apps/peppermint/docker-compose.arm64.yml:5]: Dev indicates that it's a development build so not ready to be used. Also its not the same size of the other one so we can assume this app does not support arm

Comment (steveiliop56) [reply apps/peppermint/docker-compose.arm64.yml:5]: Ok dropping support for it...

**Review by meienberger — APPROVED** [review]

**Pull request merged**